### PR TITLE
readme update :  test data for solar

### PR DESCRIPTION
**Pull request by loumir**

### CODE REVIEW

**Comment by loumir** [comment]
update readme for adding a tests data directory

File: README.md [hunk 1]
```diff
@@ -69,5 +69,6 @@ This work is licensed under a
 [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
   
 # Test data directory for solar data 
-materials from solar data community (coll. V. Delouille, B. Cecconni, etc.)
+Materials from solar data community (coll. V. Delouille, B. Cecconni, etc.)
+
 Include testdata to study mapping of ucds for solar data , and possible inclusion of new terms for sloar data collections
```
Comment (molinaro-m) [inline README.md]: sloar -> solar (typo)

Plus I wonder if it would be better to leave the License at bottom

**Review by molinaro-m — COMMENTED** [review]
small typo fix request and question on section position in the README

File: README.md (hunk 1 above)
Comment (loumir) [reply README.md]: ok , thanks , sounds all good.

**Review by BaptisteCecconi — CHANGES_REQUESTED** [review]
please fix my last name :-) (line 72)

**Review by BaptisteCecconi — APPROVED** [review]

**Pull request merged**